canonical-web-and-design/ubuntu.com
# Rebuilds /what-is-openstack page at /openstack

## Done

- Rebuilds /what-is-openstack page at /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/what-is-openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit#heading=h.c45r73q8oci4)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4134
